interfaces/portal/src/app/pages/project-payment-transfer-list/components/retry-transfers-dialog/retry-transfers-dialog.component.ts

```diff
@@ -66,14 +72,33 @@ export class RetryTransfersDialogComponent {
   }));
 
   public retryFailedTransfers({ referenceIds }: { referenceIds: string[] }) {
+    const eventName =
+      referenceIds.length === 1
+        ? 'retry-transaction:single'
+        : 'retry-transaction:multiple';
```

Comment:
these are agreed upon eventNames following some standard? Should these not come from an enum maybe?

Reply:
Yeah, its still somewhat 'in the air' what the actual standard-pattern will be;
And how much "system" we need.

These strings will almost always be 'very local', so only relevant in this specific file.

The format/syntax will very likely be repeated elsewhere, and is related to "how we (want to) get the data out" using the Matomo-reporting tools. But its not 'set in stone' yet what the best way to do this is.